.github/workflows/release.yml

```diff
@@ -62,8 +62,11 @@ jobs:
         with:
           name: share
           path: share
+          fail_on_unmatched_files: false
```

Comment:
You put it on the wrong step, it should be line 72 `softprops/action-gh-release@v2`, after the `with`.

Reply:
You also miss the zip archive line 73.